backend/ee/danswer/external_permissions/slack/utils.py

```diff
@@ -0,0 +1,18 @@
+from slack_sdk import WebClient
+
+from danswer.connectors.slack.connector import make_paginated_slack_api_call_w_retries
+
+
```

Comment:
this exists to create a map of all slack user ids to emails because the other calls return the slack user ids but we need the emails